retrofit/src/test/java/retrofit/RequestBuilderTest.java

```diff
@@ -910,6 +910,21 @@
     assertThat(request.body()).isNull();
   }
 
+  @Test public void getWithUrlAbsoluteDifferentHost() {
```

Comment:
This is already tested directly above. The base URL authority is 'example.com' and we switch it to 'example.org'. I realize they look very similar which can cause confusion. Maybe just change the above two tests to 'example2.com`?